src/lib/dnssd/Resolver_ImplNone.cpp

```diff
@@ -52,6 +53,14 @@ class NoneResolver : public Resolver
     {
         return CHIP_ERROR_NOT_IMPLEMENTED;
     }
+    CHIP_ERROR StartBrowseOperationals(Optional<uint64_t> compressedFabricIdFilter) override { return CHIP_ERROR_NOT_IMPLEMENTED; }
+    CHIP_ERROR StartBrowseCommissionables() override { return CHIP_ERROR_NOT_IMPLEMENTED; }
```

Comment:
what is the relation between these and the Discover* calls above? 

Duplicating API seems not ideal, it would be nice to only have a single path for exercising things.

Reply:
The various discover calls have been tailored to do "browse + resolve" with custom work in-between. 
The proposed API allows to separate those.